Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -46,6 +46,16 @@ public enum SolverSafeZone
             AtopPalm = 4
         }
 
+        // Subtract one here because AtopPalm is not along the palm, in clockwise direction
+        private static readonly int SolverSafeZoneClockwiseCount = System.Enum.GetValues(typeof(SolverSafeZone)).Length - 1;
+        private static readonly SolverSafeZone[] handSafeZonesClockWiseRightHand = new SolverSafeZone[] {
+            SolverSafeZone.UlnarSide,
+            SolverSafeZone.AboveFingerTips,
+            SolverSafeZone.RadialSide,
+            SolverSafeZone.BelowWrist,
+            SolverSafeZone.AtopPalm
```

Comment:
Since `AtopPalm` seems to be getting special-cased out for safe zone positioning, is it needed in this array?

Reply:
Good catch! Removed.